### ISSUE TITLE
2021 Agriculture Dataset EDA

### ISSUE THREAD

**Issue by henrylao** [akailany/Pattern-Segmentation-in-Aerial-Agricultural-Images#10]
Overview
---
Analyze the given data for Agriculture 2021 -- the paper operated on a dataset that was
imbalanced. In turn, it's loss function was developed in response to the imbalances that exist. Graphs + tables are going to be useful for presentation purposes therefore
`matplotlib, seaborn, or plotly` -- whatever plotting tool should be applied to construct visuals for this

Starter Questions 
- What does 2021 look like? 
- How many classes are there? 
- How many images for each classes?
- How many images in total (train, test, val)?

Checklist
- [ ] Answers for the questions above (more answers, if there are questions that were missed)
- [ ] Graphs & Tables describing the dataset
